README.md

```diff
@@ -295,6 +295,17 @@ var htmlReporter = require('pa11y/reporter/html');
 var html = htmlReporter.process(results, url);
 ```
 
+### Validating Actions
+
+Pa11y exposes a function which allows you to validate [action](#actions) strings before attempting to use them.
+
+This function accepts an action string and returns a boolean indicating whether the action is valid or not:
```

Comment:
I had to go back and check to try and puzzle out what you meant by "valid" 😁 

Maybe be explicit that "valid" means "specified in the actions API"